theano/tests/test_flake8.py

```diff
@@ -1,6 +1,6 @@
-"""
-Test flake8 errors.
-"""
+
```

Comment:
At the top of files, we want docstring, not comment.

Reply:
ok

Reply:
I retained the doc strings at the top now